src/themes/default/modules/toast.variables

```diff
@@ -24,10 +25,16 @@
 @toastSuccessColor: @successColor;
 
 /* Icon */
-@toastIconRightMargin: 0.6em;
+@toastIconContentPadding: 3em;
```

Comment:
Alright, I am going to fix the mentioned issues.. 🙂

Reply:
Cool, keep up the good work :)

Reply:
Wow, that was quite a headache today and took some time to solve, but i hope i covered everything now and fixed a lot:

- Every combination of toast or message with top or bottom progress is now fixed. Every different width (according to content) does not disturb any other toast or it's progres anymore
- added `compact` option (default true) This will display the toast (regardless if toast or message) in the fixed 300px width (if "compact2 is part of className it will also be considered the same way. Settting this to false will still nicely display every toast responsively, so i thought to make this option false by default (you guys can decide)
- fixed alignment for center and right positions if different toasts are shown at the same time
- removed unused opacity variables, which were left behind from the last commit
- I left out the dist css/js this time :smile: